static/home/css/style.css

```diff
@@ -240,12 +248,21 @@ a:hover{
 }
 #about-us .container div{
 	max-height: 500px;
+	align-items: center;
+	justify-content: center;
```

Comment:
I don't see any use of `justify-content property here.

Reply:
Okay I'll remove it.